build/azure/terraform-bundle.hcl

```diff
@@ -5,7 +5,7 @@
 terraform {
   required_providers {
     azurerm = {
-      version = "3.44.0"
+      version = "3.47.0"
```

Comment:
Can you also bump the version in https://github.com/gardener/terraformer/blob/f4f0154986c8ed3a7f2643749ebc6a8a5fc7f031/build/all/terraform-bundle.hcl#L10-L12?